mod_bot/main.py

```diff
@@ -187,7 +187,7 @@ def add_minutes(hstart, mstart, mn):
                 raid_info = raid_info.replace('[[lat]]', gym_data[3])
                 raid_info = raid_info.replace('[[lng]]', gym_data[4])
                 if gym_data[5] is True:
-                    raid_info = raid_info + "\n\n**Cette arène est éligible pour un raid EX**"
+                    raid_info = raid_info + "**Eligible Raid EX** : Oui"
```

Comment:
Not a fan of this one, unless there's also a "no" showing if not eligible.

Reply:
Agreed. 
I didn't want to change the logic behind it. It was mainly to make it appear near the other valuable informations, at the channel creation.

I will revert/change it back for now, but it could definitely be a nice improvement for a future update.